### ISSUE TITLE
Exiting edit-mode loses keyboard focus

### ISSUE THREAD

**Issue by barklund** [GoogleForCreators/web-stories-wp#1868]
## Bug Description

When exiting edit-mode (either by clicking outside or pressing <kbd>esc</kbd>), text element appears selected but does not have keyboard focus

## Expected Behaviour

When edit-mode exists and element still has visual focus, it should also retain keyboard focus.

## Steps to Reproduce

*By clicking outside*
* Add text element
* Click to edit
* Click outside text element to exit edit-mode
* Press <kbd>del</kbd> to delete element and observe, that nothing happens as element does not have keyboard focus.

*By pressing <kbd>esc</kbd>*
* Add text element
* Click to edit
* Press <kbd>esc</kbd> exit edit-mode
* Press <kbd>del</kbd> to delete element and observe, that nothing happens as element does not have keyboard focus.

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by miina** [comment]
Closing as this seems to work as expected on the latest master branch -- the focus seems to remain with both Esc/clicking outside.

cc @barklund
